genai-perf/genai_perf/inputs/retrievers/base_file_input_retriever.py

```diff
@@ -56,7 +56,7 @@ def _verify_file(self, filename: Path) -> None:
 
     def _get_content_from_input_file(self, filename: Path) -> Union[
         Tuple[List[str], List[str]],
-        Tuple[List[str], List[str], List[Dict[Any, Any]]],
+        Tuple[List[str], List[int], List[Dict[Any, Any]]],
```

Comment:
Is `List[int]` here timestamps?

Reply:
yes, it's timestamps

Reply:
I feel like the return type described here is very complicated. Is there any way to improve this? I see we defined some type aliases. Can we take advantage of those to give more context to the return type?

For example, 
```suggestion
        Tuple[TextData, Timestamps, OptionalData],
```